app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,13 @@
+from players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, nickname: str, favourite_dish: str, hummer_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
+
+    def get_rating(self) -> int:
+        return self._hummer_level
+
+    def player_info(self) -> str:
+        return f"Dwarf Warrior: {self.nickname}"
```

Comment:
The variable `nickname` is used in the `player_info` method, but it is not defined in the `DwarfWarrior` class. Ensure that `nickname` is properly initialized, possibly in the `Dwarf` class, which `DwarfWarrior` inherits from.